core/src/tile/tileManager.cpp

```diff
@@ -6,32 +6,30 @@
 #include "tile/mapTile.h"
 #include "view/view.h"
 
+#include "glm/gtx/norm.hpp"
+
+#include <chrono>
 #include <algorithm>
 
-TileManager::TileManager() {
+#define DBG(...)
```

Comment:
Until we have decided on a more elaborate logging mechanism, local DBG macros are handy to switch on while developing a particular feature. Can be removed now or later